src/script/hydra-eval-jobset

```diff
@@ -713,12 +724,13 @@ sub checkJobsetWrapped {
 
     my $flakeRef = $jobset->flake;
     if (defined $flakeRef) {
+        my ($ref, $attr) = split '#', $flakeRef, 2;
         (my $res, my $json, my $stderr) = captureStdoutStderr(
-            600, "nix", "flake", "metadata", "--refresh", "--json", "--", $flakeRef);
+            600, "nix", "flake", "metadata", "--refresh", "--json", "--", $ref);
         die "'nix flake metadata' returned " . ($res & 127 ? "signal $res" : "exit code " . ($res >> 8))
             . ":\n" . ($stderr ? decode("utf-8", $stderr) : "(no output)\n")
             if $res;
-        $flakeRef = decode_json($json)->{'url'};
+        $ref = decode_json($json)->{'url'};
```

Comment:
With this patch the evaluator stops picking up new changes.

After testing I found out that this line seems should stay `$flakeRef = decode_json($json)->{'url'};`.